### PR TITLE
feat(neon-js/cli/rpc): modularize getBlock

### PR DESCRIPTION
**Pull request by phetter**
- broke neon-js getBlock into CLI and module components for comosability
- added template to start neon-js-based get_last_transactions_by_address composition

### CODE REVIEW

File: src/nodejs/neon-js/modules/get_last_transactions_by_address.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  'address': address,
+  'page': program.page,
+  'time': program.time ? program.time : false,
+  'human': program.Human ? program.Human : false,
+  'index': program.index
+}
+
+if (defly) dbg.logDeep('argz: ', argz)
+
+get_last_transactions_by_address.run(argz).then((r) => {
```
Comment (phetter) [inline src/nodejs/neon-js/modules/get_last_transactions_by_address.js]: This insertion is just a temporary placeholder, will fix this use of 'argz' var in upcoming feat(neon-js/cli/rpc/): get_last_transactions_by_address PR

File: src/nodejs/neon-js/modules/getBlock.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    program.debug = false
+    program.node = ''
+    program.hash = null
+    program.time = false
+    program.human = false
+    program.index = 0
+  }
+
+  function print(msg) {
+    console.log(msg);
```
Comment (DalderupMaurice) [inline src/nodejs/neon-js/modules/getBlock.js]: It feels a little redundant to have a function for 1 console log line :P

File: src/nodejs/neon-js/modules/get_last_transactions_by_address.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+const get_last_transactions_by_address = require('nodejs_neoscan/modules/get_last_transactions_by_address')
+
+var cfg       = require('nodejs_config/config.js')
+var config    = cfg.load('nodejs_config/nodejs.config.json')
+
+let defly = false
+let address
+
+function print(msg) {
+  console.log(msg)
```
Comment (DalderupMaurice) [inline src/nodejs/neon-js/modules/get_last_transactions_by_address.js]: same thing here - If you'd like to use the function per se, I'd suggest moving it to the debug section maybe? 
Duplicate code is never good :)

File: src/nodejs/neon-js/modules/get_last_transactions_by_address.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  // summarize transaction - show amount of last n txs or similar
+  .parse(process.argv)
+
+if (program.debug) {
+  print('DEBUGGING: ' + __filename)
+  defly = true
+}
+
+if (!program.net) {
+}
```
Comment (DalderupMaurice) [inline src/nodejs/neon-js/modules/get_last_transactions_by_address.js]: empty if

File: src/nodejs/neon-js/modules/getBlock.js (hunk 2 above)
Comment (phetter) [reply src/nodejs/neon-js/modules/getBlock.js]: Noted, but I'd like to keep the print macro for now.

**Pull request merged**